pyproject.toml

```diff
@@ -1,10 +1,14 @@
 [tool.ruff]
-exclude = [
+extend-exclude = [
   "python-stdlib",
   "unix-ffi",
 ]
+line-length = 99
+target-version = "py38" # enable use of walrus operator
```

Comment:
Bumped to 3.8 to support the walrus operator which is supported and used.

```
invalid-syntax: Cannot use named assignment expression (`:=`) on Python 3.7 (syntax was added in Python 3.8)
   --> micropython/bluetooth/aioble/aioble/peripheral.py:137:20
    |
135 |                 (16, _ADV_TYPE_UUID128_COMPLETE),
136 |             ):
137 |                 if uuids := [bytes(uuid) for uuid in services if len(bytes(uuid)) == uuid_len]:
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
138 |                     resp_data = _append(adv_data, resp_data, code, b"".join(uuids))
```

I think this would need the same change on the micropython/micropython repo as well